### PR TITLE
Frontend container returns 404 for routes

### PR DESCRIPTION
**Pull request by jrw972**
Problem
-------

If a user navigates to a url (that is not the index) then the frontend container returns 404.

Solution
--------

Return the index for all routes.

### CODE REVIEW

**Comment by JackDiSalvatore** [comment]
@jrw972 
I'm running `docker run -p 80:80 libre311-frontend-image`
I go to `localhost:80` but I see a blank white screen and in the console see this

```
Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "text/html". Strict MIME type checking is enforced for module scripts per HTML spec.
```

**Comment by jrw972** [comment]
Thanks for testing this.  I think I know the problem.

**Comment by jrw972** [comment]
@JackDiSalvatore Can you review this again?

File: nginx-default.conf [hunk 1]
```diff
@@ -0,0 +1,44 @@
+server {
+    listen       80;
+    server_name  localhost;
+
+    #access_log  /var/log/nginx/host.access.log  main;
+
+    location / {
+        root   /usr/share/nginx/html;
+        index  index.html index.htm;
+        try_files $uri $uri/ /index.html /index.htm
```
Comment (JackDiSalvatore) [inline nginx-default.conf:10]: I still think you need a semi-colon here?
This works for me now when I add it in

**Review by JackDiSalvatore — CHANGES_REQUESTED** [review]

File: nginx-default.conf (hunk 1 above)
Comment (jrw972) [reply nginx-default.conf:10]: You are correct.

**Review by JackDiSalvatore — APPROVED** [review]

**Pull request merged**